integration/testdata/debug/go/.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
wonder if this should just go at the root level?

Reply:
Possibly.  For now I moved it into `integration/testdata/debug`